api/src/main/java/com/datastrato/gravitino/rel/expressions/distributions/Strategy.java

```diff
@@ -18,6 +18,11 @@
  * </ul>
  */
 public enum Strategy {
+  /**
+   * No distribution strategy. This is the default strategy. Will depend on the allocation strategy
+   * of the underlying system.
+   */
+  NONE,
```

Comment:
This brings an imcompatible change.

Reply:
If we use an elder client, it will cause error. Because we modification the value of enumeration.

Reply:
My fault.